src/views/payment-sheet-views/card-view.js

```diff
@@ -124,40 +121,46 @@ CardView.prototype._generateHostedFieldsOptions = function () {
     }
   };
 
-  if (!overrides) {
-    return options;
-  }
+  if (overrides) {
+    if (overrides.fields) {
+      if (overrides.fields.cvv && overrides.fields.cvv.placeholder) {
+        this._hasCustomCVVPlaceholder = true;
+      }
 
-  if (overrides.fields) {
-    if (overrides.fields.cvv && overrides.fields.cvv.placeholder) {
-      this._hasCustomCVVPlaceholder = true;
-    }
+      Object.keys(overrides.fields).forEach(function (field) {
+        if ((field === 'cvv' || field === 'postalCode') && overrides.fields[field] === null) {
+          delete options.fields[field];
+          return;
+        }
 
-    Object.keys(overrides.fields).forEach(function (field) {
-      if ((field === 'cvv' || field === 'postalCode') && overrides.fields[field] === null) {
-        delete options.fields[field];
-        return;
-      }
+        if (!options.fields[field]) {
+          return;
+        }
 
-      if (!options.fields[field]) {
-        return;
-      }
+        assign(options.fields[field], overrides.fields[field], {
+          selector: options.fields[field].selector
+        });
+      });
+    }
 
-      assign(options.fields[field], overrides.fields[field], {
-        selector: options.fields[field].selector
+    if (overrides.styles) {
+      Object.keys(overrides.styles).forEach(function (style) {
+        if (overrides.styles[style] === null) {
+          delete options.styles[style];
+          return;
+        }
+
+        assign(options.styles[style], overrides.styles[style]);
       });
-    });
+    }
   }
 
-  if (overrides.styles) {
-    Object.keys(overrides.styles).forEach(function (style) {
-      if (overrides.styles[style] === null) {
-        delete options.styles[style];
-        return;
-      }
+  if (!hasCVVChallenge) {
```

Comment:
We probably want people to be able to supply the cvv if it's specified in the overrides (in case they're verifying on the server).

So this could be:

```js
var hasCVVOverride;
var hasPostalCodeOverride;

// stuff

if (overrides) {
  if (overrides.fields) {
    hasCVVOverride = Boolean(overrides.fields.cvv);
    hasPostalCodeOverride = Boolean(overrides.fields.postalCode);

// stuff

if (!hasCVVChallenge && !hasCVVOverride) {
```